packages/client/src/api/forum.ts

```diff
@@ -0,0 +1,14 @@
+import { threads } from 'src/mock/forum-threads';
+import { ForumItem as ForumItemProps } from 'src/services/reducers/forum-slice';
+
+class ForumApi {
+  getForum = (
+    pageNumber: string | number = 1
+  ): Promise<Array<ForumItemProps>> => {
+    return new Promise<Array<ForumItemProps>>((resolve) => {
```

Comment:
```suggestion
  ): Promise<ForumItemProps[]> => {
    return new Promise<ForumItemProps[]>((resolve) => {
```
можно упростить синтаксис